step-framework-parent/step-framework-timeseries/src/main/java/step/core/timeseries/metric/MetricType.java

```diff
@@ -0,0 +1,37 @@
+package step.core.timeseries.metric;
+
+import step.core.accessors.AbstractIdentifiableObject;
+
+public class MetricType extends AbstractIdentifiableObject {
+    
+    private String label;
+    private String oqlQuery;
```

Comment:
@lucianbaciu The metric type should be identified by its name. In which field are we storing this information? What is the reason for the field oqlQuery? In my opinion the associated query for searching a metric should always be metricType = <metricname>

Reply:
for thread group, there is a combination of 2 filters: metricType = sampler + type = 'threadGroup'. it may be the case for other metrics maybe?

Reply:
@jeromecomte label = name. I will rename 'label' to 'name'

Reply:
I get the point. I've discussed this with @david-stephan. We would be in favour of changing the meaning of the the field metricType in the timeseries so that it **uniquely** points to the name of a defined metricType in the collection metricType. In this case it would mean:
- Create a metric type in the collection metricType for the thread group. name = executions/thread-count, unit = 1
- Change the value of the metricType accordingly at ingestion of threadgroup metrics
- To remain backward compatible you could keep the attribute type = 'threadGroup' at ingestion threadgroup metrics and only filter on this attribute when querying the data for the Thread Group chart